os.cpp

```diff
@@ -1,380 +0,0 @@
-/*
- * os.cpp
- *
- * Baker, Ballard, Jager-Kujawa
- * CSC 341
- * Spring 2016
- */
-
-#include "os.hpp"
-
-queue<User> readyQueue = queue<User>();
-queue<User> blockedQueue = queue<User>();
-User currentUser;
-bool semaphore = unlocked;
-
-int sysclock;
-int switchTime;
-
-// Dump contents of main memory and all registers and semaphore status
-void dump()
-{
-	// Dump Registers
-	User temp;
-	cout << endl << ">>>>>>DUMPING REGISTERS<<<<<<" << endl;
-	// Dump semaphore
-	cout << endl << "Semaphore status: " << \
-		(semaphore == locked ? "Locked" : "Unlocked") << endl;
-	printQueue("readyQueue", readyQueue, readyQueue.size());
-	printQueue("blockedQueue", blockedQueue, blockedQueue.size());
-
-	// Dump memory
-	cout << endl << ">>>>>>DUMPING MEMORY<<<<<<" << endl;
-	// Print table header
-	cout << "_____________________________________" \
-		<< "__________________________" << endl;
-	cout << "ad: valu | ad: valu | ad: valu | ad: " \
-		<< "valu | ad: valu | ad: valu" << endl;
-	cout << "_____________________________________" \
-		<< "__________________________" << endl;
-	// Print table contents
-	for (int i = 0; i < 256; i++) {
-		if (i % 6 == 0)
-		cout << endl;
-		else
-		cout << " | ";
-		cout << hex;
-		cout << setw(2) << i << ": " << setw(4) << main_memory[i];
-		cout << dec;
-	}
-	cout << endl << endl << ">\tEnd of dump" << endl << endl;
-}
```

Comment:
I'd like to make our output more concise if possible. Not a requirement, just a suggestion.